extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -10,6 +10,15 @@ class CommunityPageSpecialController extends WikiaSpecialPageController {
 	const MODAL_IMAGE_HEIGHT = 700.0;
 	const MODAL_IMAGE_MIN_RATIO = 0.85;
 	const DEFAULT_MODULES_MAX = 3;
+	// order of permissions is consistent with badges hierarchy in Discussions
+	const PERMISSIONS_HIERARCHY = [
```

Comment:
I would not call this 'permissions hierarchy' as it serves two purposes, describing the permission=>badge mapping AND permission hierarchy